### ISSUE TITLE
Fragment ID in URL not working in Chrome & Firefox when using ScrollSpy & web font

### ISSUE THREAD

**Issue by StephanBauwens** [twbs/bootstrap#19066]
Hi,

I have a page using the bootstrap (v3) affix:
http://data-onderwijs.vlaanderen.be/edulex/document.aspx?docid=13123
Everything works fine in 3 tested browsers chrome, firefox and IE11: the right panel "follows" perfectly the left panel when scrolling down the page.
But there is a problem when adding a hashtag in the url:
http://data-onderwijs.vlaanderen.be/edulex/document.aspx?docid=13123#7-4
In IE11 the behaviour is allways as expected so I see title 7.4 at the top of the window and the right panel is scrolled to 7.4, but in Chome (sometimes) and in firefox (allways) the page scrolls down to the bottom. You have to close the browser, restart it so the page is not in memory, paste the url into the adress space and hit return to see the wrong behaviour. Once the page is loaded, and you hit enter again for the second time, the behaviour is back normal as expected.
But my customers click from external websites to this anchor #7-4 and they arrive at the bottom of the page. I cant tell them to hit enter again of course ;-)
It's also very strange that this behavour happens only at bookmarks #7-3 and #7-4 and NOT at other bookmarks like 7-1 or 7. I dont understand this. I already tried changing the - into a . or a | but the behaviour stays the same when using those other characters.
When I disable the affix by typing data-spy="noaffix" (non existing data-spy) then the affix doenst work anymore (of course!) but the anchor to 7.4 in Chrome and firefox works again as expected.

**Comment by cvrebert** [comment]
@StephanBauwens Please make a JS Bin version of your page that's as simple as possible, and in particular doesn't use any JavaScript other that jQuery and Bootstrap.

**Comment by StephanBauwens** [comment]
It didn't work to create this behaviour in Js Bin. The JS bin page loads as expected in all browsers. I did narrow the error on my server to this:

- bootstrap scrollspy active and lato webfont in css: some anchors not found in Firefox
- bootstrap scrollspy deactivated: all anchors found
- bootstrap scrollspy active and trebuchet system font instead of lato webfont: all anchors found

So it looks like the combination of the webfont and bootstrap scrollspy make some of the anchors not found . I removed the lato webfont from my webpage as a workaround. I dont understand how the webfont in combination with the scrollspy can influence the anchors (and only some of them) in Firefox while it's working in IE11.
